pkg/async/schedule/aws/workflow_executor.go

```diff
@@ -168,6 +171,15 @@ func (e *workflowExecutor) formulateExecutionCreateRequest(
 }
 
 func (e *workflowExecutor) Run() {
+	for {
+		logger.Warningf(context.Background(), "Starting workflow executor")
+		err := e.run()
```

Comment:
and here

Reply:
and this is a layer above the async auto-retry layer right?  so if we exhaust all retries, then we chill for half an hour and then try again?

Reply:
wow half an hour is a goof, meant it to be half a minute - will update

Reply:
added a metric on channel close

Reply:
"async retry is for initializing clients a finite amount of times"